src/main/java/io/temporal/common/RetryOptions.java

```diff
@@ -66,10 +66,10 @@ public static RetryOptions merge(MethodRetry r, RetryOptions o) {
       if (o == null) {
         return null;
       }
-      return RetryOptions.newBuilder(o).validateBuildWithDefaults();
+      return o;
```

Comment:
previously we used to return a new copy.  Now we return the same instance back.  Can this cause issues?

Reply:
It is immutable, so no issues expected.